LibrariesPrototype/Presentation/Controls/LTGradientBarButton.swift

```diff
@@ -46,7 +45,7 @@ class LTGradientBarButton: LTView {
         }
     }
     
-    var durations: Durations = Durations(fadeIn: 1.0, fadeOut: 1.0, progress: 1.0) {
+    var durations = Durations(fadeIn: 1.0, fadeOut: 1.0, progress: 1.0) {
         didSet {
             updateGradientLoadingBar()
```

Comment:
repetitive call of updateGradientLoadingBar(), Maybe adding initialize method to call end of the button creation.